### ISSUE TITLE
./bin/coqc --version no longer works on non-local builds

### ISSUE THREAD

**Issue by JasonGross** [coq/coq#11254]
#### Description of the problem

This breaks my packaging scripts.

```
$ ./configure -prefix "$HOME/.local64/coq/coq-8.11+beta1/"
You have OCaml 4.06.1. Good!
You have OCamlfind 1.8.1. Good!
You have native-code compilation. Good!
You have the Num library installed. Good!
LablGtk3 not found:
=> no CoqIde will be built.

  Architecture                : Linux
  Sys.os_type                 : Unix
  Coq VM bytecode link flags  : -dllib -lcoqrun -dllpath /home/jgross/.local64/coq/coq-8.11+beta1//lib/coq/kernel/byterun
  Other bytecode link flags   :
  OCaml version               : 4.06.1
  OCaml binaries in           : /home/jgross/.opam/4.06.1+fp/bin/
  OCaml library in            : /home/jgross/.opam/4.06.1+fp/lib/ocaml
  OCaml flambda flags         :
  Native dynamic link support : true
  CoqIde                      : no
  Documentation               : None
  Web browser                 : firefox -remote "OpenURL(%s,new-tab)" || firefox %s &
  Coq web site                : http://coq.inria.fr/
  Bytecode VM enabled         : true
  Native Compiler enabled     : true

  Paths for true installation:
  - the Coq binaries will be copied in /home/jgross/.local64/coq/coq-8.11+beta1//bin
  - the Coq library will be copied in /home/jgross/.local64/coq/coq-8.11+beta1//lib/coq
  - the Coqide configuration files will be copied in /home/jgross/.local64/coq/coq-8.11+beta1//etc/xdg/coq
  - the Coqide data files will be copied in /home/jgross/.local64/coq/coq-8.11+beta1//share/coq
  - the Coq man pages will be copied in /home/jgross/.local64/coq/coq-8.11+beta1//share/man
  - the Coq documentation will be copied in /home/jgross/.local64/coq/coq-8.11+beta1//share/doc/coq
  - the Coqdoc LaTeX files will be copied in /home/jgross/.local64/coq/coq-8.11+beta1//share/texmf/tex/latex/misc

If anything is wrong above, please restart './configure'.

*Warning* To compile the system for a new architecture
          don't forget to do a 'make clean' before './configure'.
$ make states -j6
[...]
$ ./bin/coqc --version
Error: cannot guess a path for Coq libraries; please use -coqlib option

$ ./bin/coqc --print-version
Error: cannot guess a path for Coq libraries; please use -coqlib option


```

Consider 8.10.2 by contrast:
```
$ ./configure -prefix "$HOME/.local64/coq/coq-8.10.2/"
You have OCaml 4.06.1. Good!
You have OCamlfind 1.8.1. Good!
You have native-code compilation. Good!
You have the Num library installed. Good!
LablGtk3 not found:
=> no CoqIde will be built.

  Architecture                : Linux
  Sys.os_type                 : Unix
  Coq VM bytecode link flags  : -dllib -lcoqrun -dllpath /home/jgross/.local64/coq/coq-8.10.2//lib/coq/kernel/byterun
  Other bytecode link flags   :
  OCaml version               : 4.06.1
  OCaml binaries in           : /home/jgross/.opam/4.06.1+fp/bin/
  OCaml library in            : /home/jgross/.opam/4.06.1+fp/lib/ocaml
  OCaml flambda flags         :
  Native dynamic link support : true
  CoqIde                      : no
  Documentation               : None
  Web browser                 : firefox -remote "OpenURL(%s,new-tab)" || firefox %s &
  Coq web site                : http://coq.inria.fr/
  Bytecode VM enabled         : true
  Native Compiler enabled     : true

  Paths for true installation:
  - the Coq binaries will be copied in /home/jgross/.local64/coq/coq-8.10.2//bin
  - the Coq library will be copied in /home/jgross/.local64/coq/coq-8.10.2//lib/coq
  - the Coqide configuration files will be copied in /home/jgross/.local64/coq/coq-8.10.2//etc/xdg/coq
  - the Coqide data files will be copied in /home/jgross/.local64/coq/coq-8.10.2//share/coq
  - the Coq man pages will be copied in /home/jgross/.local64/coq/coq-8.10.2//share/man
  - the Coq documentation will be copied in /home/jgross/.local64/coq/coq-8.10.2//share/doc/coq
  - the Coqdoc LaTeX files will be copied in /home/jgross/.local64/coq/coq-8.10.2//share/texmf/tex/latex/misc

If anything is wrong above, please restart './configure'.

*Warning* To compile the system for a new architecture
          don't forget to do a 'make clean' before './configure'.
$ make states -j6
[...]
$ ./bin/coqc --version
The Coq Proof Assistant, version 8.10.2 (December 2019)
compiled on Dec 6 2019 23:12:12 with OCaml 4.06.1
$ ./bin/coqc --print-version
8.10.2 4.06.1
```

#### Coq Version

8.11+beta1

**Comment by Zimmi48** [comment]
Looks like it could be related to #10246 (cc @herbelin @ejgallego).

**Comment by herbelin** [comment]
This is indeed related to #10246. See a fix at #11254.
